okio/jvm/src/main/java/okio/ByteString.kt

```diff
@@ -141,15 +142,8 @@ internal actual constructor(
    */
   actual open fun toAsciiUppercase(): ByteString = commonToAsciiUppercase()
 
-  // TODO move substring() when https://youtrack.jetbrains.com/issue/KT-22818 is fixed
-
-  /**
```

Comment:
We should keep the KDoc in JVM module for now cause Dokka is broken. They plan to fix this in the next release, then we can move all KDoc into common.

Reply:
Sorry, missed this comment before I merged.